tests/src/Kernel/fixtures/markup_rendering_patterns/links_block.yml

```diff
@@ -0,0 +1,120 @@
+links_block_vertical_gray:
+  render:
+    '#type': pattern
+    '#id': links_block
+    '#fields':
+      settings:
+        background: "gray"
+        direction: "vertical"
+      title: 'Title Link Block'
+      links:
+        - label: 'Link number one'
+          path: 'https://example.com'
+        - label: 'Link number two'
+          path: 'https://google.com'
+        - label: 'More channels'
+          path: 'https://europa.eu'
+  assertions:
+    count:
+      'div.bg-light.px-4.py-3': 1
+      'h5.fw-bold.pb-3.mb-3.border-bottom': 1
+      'ul.ps-0.mb-0': 1
+      'li.list-unstyled': 3
+      'li.d-inline': 0
+      'li.list-unstyled.me-4-5': 2
+      'a.pb-3.d-inline-block': 3
+    equals:
+      'h5': "Title Link Block"
+      'ul li:nth-child(1) a[href="https://example.com"]': "Link number one"
+      'ul li:nth-child(2) a[href="https://google.com"]': "Link number two"
+      'ul li:nth-child(3) a[href="https://europa.eu"]': "More channels"
```

Comment:
@brummbar @drishu  I see what you mean with this but I put all the classes from netlify with the intention that if is there any change on the BCL we can detect because every release of BCL we have to search every litle change they did on the patterns. I try to put always all classes so any change we can detect also passing our tests.

Reply:
It's not the scope of this test to assert such changes though, the scope here is to make sure we are using the correct templates to render the pattern.